app/views/components/input_textarea.scala.html

```diff
@@ -23,7 +23,7 @@
     hint: Option[String] = None
 )(implicit messages: Messages)
 
-<div class="form-group @if(field.hasErrors){form-group-error}">
+<div class="form-field @if(field.hasErrors){form-field--error}">
```

Comment:
Please, double check if the name of the class contains double or single dash ("--").

Reply:
Double :)